vllm/entrypoints/openai/rpc/client.py

```diff
@@ -18,6 +18,7 @@
                                          VLLM_RPC_ZMQ_HWM, RPCAbortRequest,
                                          RPCGenerateRequest, RPCUtilityRequest)
 # yapf: enable
+from vllm.envs import VLLM_RPC_GET_DATA_TIMEOUT_MS
```

Comment:
Can we then delete the other variables:
- `VLLM_RPC_HEALTH_TIMEOUT_MS`
- `VLLM_RPC_SERVER_START_TIMEOUT_MS`

Reply:
👍 , replaced with this one timeout